src/components/modal/addtostudio/presentation.jsx

```diff
@@ -37,18 +37,19 @@ const AddToStudioModalPresentation = ({
 
     return (
         <Modal
+            standardSizes
```

Comment:
I suggest renaming `standardSizes` to `useStandardSizes` or something similar, to make it more clear that it is a boolean, and not some kind of object that contains info about sizes.

Reply:
oh right good catch!